Added support for WAP UR6,7,8 & removed invoke-command

DdenBraver opened the pull request:
The current version does not include the productID's for WAP UR6 and UR7.
It also uses invoke-command at multiple entries which may give errors authenticating, even when splitting roles between multiple servers.

vors committed:
Merge pull request #1 from PowerShell/dev

Update to version 1.1.4.0 from PSGallery

DdenBraver committed:
Adjusted DSC Resources for customer ASP4ALL & removed invoke-command due to authentication issues

DdenBraver committed:
updated code and example

DdenBraver committed:
updated code and example

DdenBraver committed:
added WAP UR7 support

msftclas commented:
Hi __@Darkentrophy__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

msftclas commented:
__@Darkentrophy__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

KarolKaczmarek commented:
@Darkentrophy Thanks for your contribution. Could you please resolve conflicts?
```
Step 1: From your project repository, check out a new branch and test the changes.

git checkout -b Darkentrophy-dev dev
git pull https://github.com/Darkentrophy/xAzurePack.git dev

Step 2: Merge the changes and update on GitHub.

git checkout dev
git merge --no-ff Darkentrophy-dev
git push origin dev
```

Darkentrophy commented:
@KarolKaczmarek I will take a look at it soon.
I am currently using a renamed (cAzurePack) version in my own environment, I will see if I can rename it back and push it before the end of this week.

msftclas commented:
__@DdenBraver__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

DdenBraver committed:
script update for support UR8

DdenBraver committed:
updated azurepackupdate, version & readme.md

DdenBraver committed:
updated examplescript and readme

DdenBraver committed:
Merge branch 'Darkentrophy-dev' into dev

DdenBraver committed:
Merge branch 'dev'

DdenBraver committed:
Merge branch 'master' into dev

DdenBraver committed:
resolve conflicts

DdenBraver committed:
minor bugfixes after conflict resolve

DdenBraver committed:
Merge branch 'ddb-powershell-dev' into dev

DdenBraver committed:
resolved unicode & tabs issue

DdenBraver committed:
added dbuser to meta.mof for xazurepackadmin

DdenBraver commented:
@KarolKaczmarek I updated the pull request with support for the released UR8 and managed to resolve the conflicts. Please let me know if you have any questions.

DdenBraver committed:
created function for connection string, added KEY values to schemas

DdenBraver commented:
@TravisEz13 I updated the keys with comments to make them more unique. I also added a function for the connectionstring in the xAzurePackAdmin resource.

DdenBraver committed:
corrected readme to UNRELEASED
